torch/distributed/__init__.py

```diff
@@ -23,7 +23,10 @@ def is_available():
         from .rpc import _init_rpc
         from .rpc import *  # noqa: F401
 
-        def init_model_parallel(worker_name, rpc_backend=RpcBackend.PROCESS_GROUP):
+        def init_model_parallel(rpc_backend=RpcBackend.PROCESS_GROUP,
+                                self_name=None,
```

Comment:
any reason for making `self_name` an optional arg?

Reply:
Thanks for pointing out, this is what caused the test failures. Reverting this.